bootloader/src/pyi_pythonlib.c

```diff
@@ -637,7 +637,7 @@ pyi_pylib_install_zlib(ARCHIVE_STATUS *status, TOC *ptoc)
      */
     archivename_obj = PI_PyUnicode_DecodeFSDefault(status->archivename);
 #endif
-    zlib_entry = PI_PyUnicode_FromFormat("%U?%d", archivename_obj, zlibpos);
+    zlib_entry = PI_PyUnicode_FromFormat("%U?%zu", archivename_obj, zlibpos);
```

Comment:
`%zu` is the formatting code for formatting `size_t` arguments (as opposed to `%d`, which is used for signed integers). So this change is there to be consistent with the change of `zlibpos` type from `int` to `size_t`.

Reply:
It's also there to future-proof things a bit, as it will be needed if/when we switch the offset formats in archive from signed to unsigned ints. (Although we will probably need to change it again to `%lli` or `%llu` if/when we go for 64-bit ints, but that's even further down the road...).

Reply:
Alright, makes sense and looks good!